Automattic/wp-calypso
# GH Actions: Create ETK release upon tagging

#### Changes proposed in this Pull Request

Upon adding a tag that starts with `editing-toolkit@`, automatically create a release draft and attach the ETK zip as a release assets.

Asking about the feasibility of changing the source of plugin releases for Atomic here: p9o2xV-1aS-p2 

#### Screenshot

([from my own Calypso fork](https://github.com/ockham/wp-calypso/releases), used for testing this PR:)

![image](https://user-images.githubusercontent.com/96308/101170301-8dcfa500-363e-11eb-9d84-b07f2c6f1755.png)

#### Testing instructions

Pretty much analoguous to https://github.com/WordPress/gutenberg/pull/27488:

- Make sure you have your own fork of this repo, and that you've added it locally as a remote (E.g. `git remote add ockham git@github.com:ockham/wp-calypso.git` if your GH username is `ockham` :slightly_smiling_face: ).
- Check out this PR's branch, and push the contents to your fork's `trunk` branch (you can reset later):
```
git push ockham add/gh-action-create-etk-release:trunk
```
- Add an ETK version tag (must start with `editing-toolkit@`), e.g. `git tag editing-toolkit@2.8.12-rc.1`.
- Push that tag to your fork: `git push --tags ockham`.
- Go to the 'Actions' tab of your fork's GH page (e.g. https://github.com/ockham/wp-calypso/actions).
- Locate the action whose description reads "Create release draft and attach asset", and click on it.
- It should take less than 10 minutes to build the plugin zip. You can watch that page as the action progresses.
- Eventually, a release draft will be created, and the plugin zip will be attached. You'll find it under 'Releases' (e.g. https://github.com/ockham/wp-calypso/releases)
